### ISSUE TITLE
Question about transposed rank metric

### ISSUE THREAD

**Issue by yuxin212** [altoslabs/perturbench#8]
Hi there, 

I recently read your interesting work. But I have a question about the transposed rank metric (eq. 3). Should I use expression of other perturbation $x_j$ from test set only or training set only or the whole set? If I only use $x_j$ from test set, the number of perturbations and expressions will be limited to the test set only. But please correct me if I am wrong, if I use training set or the whole set, I would think this could lead to data leakage. 
Thanks in advance!
